packages/react-server-cli/src/compileClient.js

```diff
@@ -0,0 +1,222 @@
+import webpack from "webpack"
+import path from "path"
+import mkdirp from "mkdirp"
+import fs from "fs"
+import ExtractTextPlugin from "extract-text-webpack-plugin"
+
+// commented out to please eslint, but re-add if logging is needed in this file.
```

Comment:
You can avoid the "uncomment to use" by suppressing the eslint no-unused-vars rule for just these two lines with an eslint enable/disable block, if you prefer: http://eslint.org/docs/user-guide/configuring#configuring-rules